test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -612,27 +581,25 @@ public void testInitVerifyColumnNames_modeOfOperation() throws Exception {
     Map<String, String> moreEntries = new HashMap<String, String>();
     moreEntries.put("db.modeOfOperation", "contentColumn");
     moreEntries.put("db.modeOfOperation.contentColumn.columnName", "blob");
+    moreEntries.put("db.uniqueKey", "table_col:int");
```

Comment:
This table (\"must be set\", table_col) was a paean to createStandardConfig. I think copying the table, db.uniqueKey, and db.singleDocContentSql from _metadataColumns, just above, would be an improvement.